src/CDSAPI.jl

```diff
@@ -37,6 +37,16 @@ function retrieve(name, params, filename)
     data
 end
 
-py2ju(dictstr::String) = JSON.parse(dictstr)
+function py2ju(dictstr::AbstractString)
+    dictstr_cpy = replace(dictstr, "'" => "\"")
+    lastcomma_pos = findlast(",", dictstr_cpy).start
```

Comment:
Could you please add a comment on this last comma issue? I couldn't grasp it by just reading the code. You mean that the CDS service sometimes produces a leading comma at the end of the dict?

Reply:
The CDS website always provides a `dict` which contains a comma at the end of the last key-value pair (the comma just after `'02'`). JSON.parse will mark this as an unformatted JSON object:
```julia
{
        'format': 'zip',
        'variable': 'surface_air_temperature',
        'product_type': 'climatology',
        'month': '02',
}
```

Reply:
Nice. Thanks for clarifying. The code could be cleaned up a bit more, but we don't want to waste too much time into an endless beautification (even though it is fun =D )